### PR TITLE
STEAMCONTROLLER_NO_SWIZZLING and TestApp update.

### PR DESCRIPTION
**Pull request by ToddLa**
added STEAMCONTROLLER_NO_SWIZZLING build options.
made the TestApp work with GCControllers and SteamControllers.

### CODE REVIEW

File: SteamController/SteamControllerManager.h [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 #import <Foundation/Foundation.h>
 
+#define STEAMCONTROLLER_NO_SWIZZLING
+#define STEAMCONTROLLER_NO_PRIVATE_API
+
```
Comment (zydeco) [inline SteamController/SteamControllerManager.h]:
```suggestion
```

don't enable these options by default, it should be possible to enable/disable them in build settings so it works with the `no-private-api` pod subspec

**Review by zydeco — CHANGES_REQUESTED** [review]
thanks for your contribution!

please remove the defines so the options aren't always enabled

**Comment by ToddLa** [comment]
Ok will do! I will also do it in PR #8 too. (You should look at just taking #8)

**Comment by ToddLa** [comment]
I updated PR #7 and PR #8

**Comment by zydeco** [comment]
Closing this since it was merged in #8

**Pull request closed without merging**